docs/data/material/pages.ts

```diff
@@ -129,34 +129,19 @@ const pages = [
         ],
       },
       {
-        pathname: '/x/react-data-grid',
-        subheader: 'data-grid',
+        pathname: '/mui-x', // the pathname does not matter here because the links to MUI X are outbound.
+        subheader: 'MUI X',
         children: [
-          { pathname: '/x/react-data-grid/', title: 'Overview' },
-          { pathname: '/x/react-data-grid/demo/', title: 'Demo' },
-          { pathname: '/x/react-data-grid/getting-started/', title: 'Getting Started' },
+          { pathname: '/x/react-data-grid', title: 'Data Grid' },
+          { pathname: '/x/react-date-pickers', title: 'Date/Time Pickers' },
         ],
       },
       {
         pathname: '/material-ui',
         subheader: 'lab',
         children: [
           { pathname: '/material-ui/about-the-lab', title: 'About the lab 🧪' },
-          {
-            pathname: '/material-ui/pickers',
-            subheader: 'pickers',
-            title: 'Date / Time',
-            children: [
-              { pathname: '/material-ui/pickers', title: 'Introduction' },
-              { pathname: '/material-ui/react-date-picker' },
-              {
-                pathname: '/material-ui/react-date-range-picker',
-                title: 'Date Range Picker ⚡️',
-              },
-              { pathname: '/material-ui/react-date-time-picker' },
-              { pathname: '/material-ui/react-time-picker' },
-            ],
-          },
+          { pathname: '/material-ui/deprecated-pickers', title: 'Date/Time Pickers', legacy: true },
```

Comment:
```suggestion
          { pathname: '/material-ui/deprecated-pickers', title: 'Date/Time Pickers'},
```

I would remove it — same reasoning as in https://github.com/mui/material-ui/pull/31810#discussion_r826984823.